public/index.html

```diff
@@ -36,5 +37,6 @@
         });
       }
     </script>
+    <script src="https://use.fontawesome.com/058fa4a144.js"></script>
```

Comment:
Not really; it was far simpler to do it that way than to wrestle with Material-UI's frustrating font api.